### PR TITLE
Update edit command testv1.2b

### LINKED ISSUE #79: Fix bug with edit command
Bug with editing people:
Screenshot 2024-03-20 at 8 49 58 PM

### PR DESCRIPTION
**Pull request by Jayne1010**
Closes #79

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -38,45 +43,46 @@ public class EditCommand extends Command {
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the person identified "
             + "by the index number used in the displayed person list. "
             + "Existing values will be overwritten by the input values.\n"
-            + "Parameters: INDEX (must be a positive integer) "
+            + "Parameters: IC (Personal Identification Number) "
```
Comment (drustanyjt) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: change to National Registration Identity Card (NRIC) instead of PIN?

**Review by drustanyjt — CHANGES_REQUESTED** [review]
Got a comment about using IC (PIN), the rest LGTM

**Review by drustanyjt — APPROVED** [review]
LGTM

**Comment by tengcharmaine** [comment]
closes #42 also.

**Pull request merged**